media-libs/opencolorio/opencolorio-1.0.9_p20170719.ebuild

```diff
@@ -0,0 +1,86 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+# Compatibility with Python 3 is declared by upstream, but it is broken in fact, check on bump
+PYTHON_COMPAT=( python{2_7,3_4,3_5,3_6} )
+
+inherit cmake-utils python-single-r1 vcs-snapshot
+
+DESCRIPTION="A color management framework for visual effects and animation"
+HOMEPAGE="http://opencolorio.org/"
+
+MY_GIT_COMMIT="7e7f10afce97254cdc1295a3594ddd5e6fb555d6"
+SRC_URI="https://github.com/imageworks/OpenColorIO/archive/${MY_GIT_COMMIT}.tar.gz \
+		-> ${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="doc opengl pdf python cpu_flags_x86_sse2 test"
+
+RDEPEND="
+	opengl? (
+		media-libs/lcms:2
+		media-libs/openimageio
+		media-libs/glew:=
+		media-libs/freeglut
+		virtual/opengl
+	)
+	python? ( ${PYTHON_DEPS} )
+	>=dev-cpp/yaml-cpp-0.5
+	dev-libs/tinyxml"
+
+DEPEND="${RDEPEND}
+	virtual/pkgconfig
+	doc? (
+		pdf? ( dev-python/sphinx[latex,${PYTHON_USEDEP}] )
+		!pdf? ( dev-python/sphinx[${PYTHON_USEDEP}] )
+	)"
+
+# Documentation building requires Python bindings building
+REQUIRED_USE="doc? ( python )"
```

Comment:
`REQUIRED_USE="doc? ( python ) python? ( ${PYTHON_REQUIRED_USE} )"` (also move it directly below `IUSE`)